lib/service_check.pm

```diff
@@ -234,6 +234,8 @@ sub install_services {
             if (is_sle($support_ver, $hdd_base_version)) {
                 if ($hdd_base_version eq '11-SP4') {
                     $service_type = 'SystemV';
+                    # Enable IPv6 forwarding on sle11sp4
+                    script_run('echo 1 > /proc/sys/net/ipv6/conf/all/forwarding') if ($srv_pkg_name eq 'radvd');
```

Comment:
why we need enable forwarding for ipv6?

Reply:
if we do not enable IPv6 forward，the service start will failed with "radvd: IPv6 forwarding seems to be disabled, exiting"

Reply:
I feel you'd better move this line to radvd.pm before service start as a preparation for service start, else here in service_check will get more and more complex.

Reply:
maybe we can put this logic into common_service_action?